packages/svelte/README.md

```diff
@@ -51,52 +49,48 @@ Please refer to the [beta docs for `react-router@6.4`][rr-beta-docs] for referen
   ];
 
   // Provide a fallbackElement to be displayed during the initial data load
-  const fallbackElement = () => h("p", "Loading..."),
+  const fallbackElement = "<p>loading...</p>";
 </script>
 
-<template>
-  <DataBrowserRouter :routes="routes" :fallbackElement="fallbackElement" />
-</template>
+<DataBrowserRouter {routes} {fallbackElement} />
```

Comment:
Man it's so cool to see in diff form just how little changes between the Vue and Svelte implementations.  I've felt that as I worked on the React/Vue implementations but hadn't yet seen it in diff form :)